main.py

```diff
@@ -4,28 +4,27 @@
 from common.utils import ynab_api
 from common.utils import config_api
 from enums import AccountTypeToProcessor
-from enums import AccountTypes
-from files_processors.raw_files import RawCSVFile
 
-CONFIG_FILE_PATH = "config.json"
+CONFIG_FILE_PATH = "~/code/ynab-csv-converter/config.json"
```

Comment:
relative path does not work here ?

Reply:
The relative path doesn't do well anymore when you set the main.js script as a shell script alias. You can run it from everywhere